scripts/install-windows-prereqs.ps1

```diff
@@ -543,11 +561,13 @@ function Install-DCAP-Dependencies {
                     'path'        = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel*SGX*DCAP*\base\WindowsServer2019_Windows10"
                     'location'    = 'root\SgxLCDevice'
                     'description' = 'Intel(R) Software Guard Extensions Launch Configuration Service'
+                    'hardwareId'  = '*INT0E0C'
                 }
                 'sgx_dcap' = @{
                     'path'        = "$PACKAGES_DIRECTORY\Intel_SGX_DCAP\Intel*SGX*DCAP*\dcap\WindowsServer2019_Windows10"
                     'location'    = 'root\SgxLCDevice_DCAP'
                     'description' = 'Intel(R) Software Guard Extensions DCAP Components Device'
+                    'hardwareId'  = 'SWC\VEN_INT&DEV_0E0C_DCAP'
```

Comment:
This change and the change on line 564 should be applied to the Win10 configuration from lines 573-584 as well.